hamper/plugins/questions.py

```diff
@@ -39,6 +39,7 @@ def setup(self, *args):
             ("I'm sorry, I was thinking of bananas", 0.03),
         ]
 
+        responses += [(x, 'eq/10') for x in (list(open('oblique.txt')))]
```

Comment:
I think this will make the `eq` probability something like 1 in 100, making bananas too likely. Lets change 0.03 on bananas to something like `eq/20`?